src/sentry_plugins/twilio/plugin.py

```diff
@@ -73,6 +83,9 @@ class TwilioConfigurationForm(forms.Form):
 
     def clean_sms_from(self):
         data = self.cleaned_data["sms_from"]
+        if is_sender_alphanumeric_code(data):
+            if not validate_alphanumeric_sender(data):
+                raise forms.ValidationError(f"{data} is not a valid sender ID.")
```

Comment:
Could this be

`if is_sender_alphanumeric_code(data) and not validate_alphanumeric_sender(data):`?

Reply:
Thank you, yes, changed the nested if's to a single line 😊